CHANGELOG.md

```diff
@@ -2,9 +2,12 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Rename `openstack.ClusterAlterGroup` to be `openstack.ClusterAlertGroup`
```

Comment:
shouldn't this be rancher2?

Reply:
and that's why multi-tasking is bad... :D